mutacc/cli/importing.py

```diff
@@ -0,0 +1,56 @@
+import json
+import sys, os
+
+import click
+
+from mutacc.utils.hk_call import find_files
+
+@click.command('import')
+@click.option('--bam', multiple = True, type = click.Path(exists = True), help = 'path to .bam file')
+@click.option('--fastq', multiple = True, type = click.Path(exists = True), help = 'path to .fastq file')
+@click.option('--variant', help = 'Variant data as JSON string, or path to .json file')
+@click.option('--case', help = 'Metadata for the case as JSON string or path to .json file')
+@click.pass_context
+def importing(context,bam,fastq,variant,case):
```

Comment:
How about spaces between the parameters i.e. "context, bam". Makes it easier to read

Reply:
I agree, that is a standard that we use, should look like:
```python
def importing(context, bam, fastq, variant, case):
```